src/org/parosproxy/paros/view/SiteMapPanel.java

```diff
@@ -390,6 +391,19 @@ public void reloadContextTree() {
 		this.contextTree.nodeStructureChanged(root);
 	}
 
+	public Context getSelectedContext() {
+	    JTree treeContext = getTreeContext();

```

Comment:
- This statement could be removed (`treeContext` is already initialised).
 - There are some mixed indentation chars.